src/components/icon/demoBasicUsage/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
```

Comment:
I removed as much of the CSS as I think I can.  Let me know if there's more I can do.